### DISCUSSION TITLE
Default fields and field groupings/locations - what do the majority need?

### DISCUSSION THREAD

**Discussion by mikeallbutt** [DiscipleTools/disciple-tools-theme#1147] [Q&A]
As a user of DT for quite some time, I have made a few adjustments to suit my needs when the default DT was in my opinon lacking.

I wanted to share in this chat my modifications and hear from others their thoughts and see if some of these modifications need to be included in the default DT setup.

In my research prayer is a key in a contact going forward. I hence made `prayer` as a new milestone in the faith milestones area and log it when it is known that the contact has been praying. (they have a prayer life)

Another thing is that I moved was the baptism date out of the details tile and into the faith tile just before who baptised the contact.

As I have been using the new theme I wondered why the `faith tile` and `status tile` are not the same one tile. Doing so would simplify things - considering pretty the things that record are similar.

I would also merge the `details tile` with the `name tile`and have the type of contact information to be in the placed in the details tile and have that details tile be first tile offered, (before the status one) but use the minimiser to show or hide the details information if desired. 

What do others think and do?

**Comment by ahillbilly** [comment]
great suggestions

**Reply by ahillbilly** [reply]
From Facebook status should be at the top of the list, not the bottom of the status list by default.
